CONTRIBUTING.md

```diff
@@ -1,26 +1,16 @@
-# Web Media Text Tracks Community Group
+# Contributor License Agreement
 
-This repository is being used for work in the Web Media Text Tracks Community Group, governed by the [W3C Community License
-Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/). To contribute, you must join
-the CG.
+Contributions to this repository are intended to become part of Recommendation-track documents governed by the W3C Patent Policy and Document License. To contribute, you must either participate in the [Timed Text Working Group](https://www.w3.org/AudioVideo/TT/) or make a non-member patent licensing commitment.
```

Comment:
You don't have to participate in the TTWG to make a contribution and the CLA above already makes a non-member patent licensing commitment.

There is far less legal conflict between the way that the CG works and the way the WGs work than you think. I disagree with making this a TTWG-only repository.

Reply:
> You don't have to participate in the TTWG to make a contribution and the CLA above already makes a non-member patent licensing commitment.

The same is true for IMSC: anyone can make a contribution, bound to the W3C CLA.